### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by LIBERCOSOFT**
# In this PR;
- Create a filled-screen pop-up menu with Javascript
- Added a custom hamburger menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,11 +28,33 @@
...
+                <li>
+                  <a href="#intro-headline" class="cancel-menu">Hello</a>
+                </li>
+                <li>
+                  <a href="#projects-section" class="cancel-menu">Portfolio</a>
+                </li>
+                <li><a href="#about-section" class="cancel-menu">About</a></li>
+                <li><a href="#contact-me" class="cancel-menu">Contact</a></li>
+              </ul>
+            </div>
```
Comment (tresorsawasawa) [inline index.html:56]: - [ ] Kindly make sure the menu follows the design from [your chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g. all mobile menu options must be centered,...).

| Figma design | Your Design |
| ------------------- | ---------------- |
| Screenshot from 2022-09-06 02-03-15 | Screenshot from 2022-09-06 02-06-53 |

File: script.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  dropdown.classList.toggle('visibility');
+  nameTag.classList.toggle('pauseName');
+});
+
+for (let i = 0; i < cancelMenu.length; i += 1) {
+  cancelMenu[i].addEventListener('click', () => {
+    dropdown.classList.toggle('visibility');
+    nameTag.classList.toggle('pauseName');
+  });
+}
```
Comment (tresorsawasawa) [inline script.js:16]: - [x] **In the README.md File:** Kindly make sure you add all the contributor's authors with their social links in your README to make it more professional.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @LIBERCOSOFT 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @LIBERCOSOFT  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**